<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormKabupaten</class>
 <widget class="QDialog" name="FormKabupaten">
  <property name="windowTitle">
   <string>Form Kabupaten</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label1">
     <property name="text">
      <string>Nama Kabupaten</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="inputNama"/>
   </item>
   <item>
    <widget class="QLabel" name="label2">
     <property name="text">
      <string>Provinsi</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboProvinsi"/>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="btnSimpan">
       <property name="text">
        <string>Simpan</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBatal">
       <property name="text">
        <string>Batal</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
</ui>
